tests/php/phpstan/phpstan.neon

```diff
@@ -0,0 +1,15 @@
+parameters:
+  reportUnmatchedIgnoredErrors: false
+  bootstrap: /web/module/tests/php/phpstan/bootstrap.php
+  paths:
+    - /web/module/src
+  ignoreErrors:
+    # module specific
+    - '~Constant _THEME_COL_DIR_ not found.~'
+    - '~Iterating over an object of an unknown class mysqli_result\.~'
```

Comment:
are these errors are really needed here?

Reply:
I try to remove all for checking